docker_plugin/utils.py

```diff
@@ -206,7 +174,51 @@ def get_container_dictionary(client, ctx):
             return None
 
 
-def get_start_params(ctx):
+def get_remove_container_params(container_id, params, ctx):
+    """ Maintain the list of supported parameters for the docker API
+        remove_container function. use this with get_params.
+
+    :param container_id: the container_id.
+    :param params: The parameters given in the inputs section
+        to the remove_container operation.
+    :param ctx: the cloudify context.
+    returns the dictionary returned from get_params.
+    """
+
+    supported_params = \
+        ['container', 'v', 'link', 'force']
+
+    arguments = get_params(supported_params)
+    arguments['container'] = container_id
```

Comment:
overriding container_id - either make the user's input be the overriding value, or raise an error if the user passes container_id